app-emulation/retroarch/metadata.xml

```diff
@@ -13,4 +9,8 @@
 	<use>
 		<flag name="cg">Use NVIDIA CG Toolkit for CG shaders.</flag>
 	</use>
+	<maintainer type="person">
```

Comment:
please declare gentoo-proxy-maintainers project last, as I told you before

Reply:
Done.